tests/Unit/PointwiseFunctions/AnalyticData/NewtonianEuler/KhInstability.py

```diff
@@ -0,0 +1,48 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def mass_density(x, adiabatic_index, strip_bimedian, strip_thickness,
+                 strip_density, strip_velocity, background_density,
+                 background_velocity, pressure, perturbation_amplitude,
+                 damping_factor):
+    return strip_density \
```

Comment:
[optional] I personally would find this code easier to read if instead of a ternary you did:
```python
if ...:
  return blah
return bla
```
and for `specific_internal_energy` too